### PR TITLE
feat: Audio recorder

### PR DESCRIPTION
**Pull request by heyumoAi**
Description: Add dtk audio recorder interface

Log: dtk audio recorder interface

### CODE REVIEW

**Review by Groveer — COMMENTED** [review]
/+1

**Review by xzl01 — APPROVED** [review]

**Review by dandan-y — COMMENTED** [review]
/+1

File: src/multimedia/audiorecorder/daudiorecorder_p.h [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        : QObject(parent), q_ptr(parent)
+    {
+    }
+
+signals:
+
+public slots:
+
+private:
+    DAudioRecorder *q_ptr;
```
Comment (toberyan) [inline src/multimedia/audiorecorder/daudiorecorder_p.h:28]: 指针未初始化

**Pull request merged**